### PR TITLE
Feature/201 new design

### LINKED ISSUE #201: Add whitelabelling to cas page
- Add whitelabelling
- Apply new design

### PR DESCRIPTION
**Pull request by sklein94**
Resolves #201

### CODE REVIEW

File: app/src/main/resources/custom_messages.properties [hunk 1]
```diff
@@ -34,4 +38,21 @@ pwdMustContainLowerCaseLetter=At least 1 lower case letter must be included
...
\ No newline at end of file
+pwdCharacters=characters
+
+pwd.change.default.error=The password could not be changed.
+
+# Generic Success Screen Messages
+screen.success.header=Log In Successful
+screen.success.success=You have successfully logged in.
+screen.success.security=For security reasons, you should <a href="logout" class="text-brand focus-visible:ces-focused">log out</a> again when leaving the password-protected area.
+screen.logout.header=Abmeldung erfolgreich
```
[the remark this answers is not in the document]
Comment (sklein94) [reply app/src/main/resources/custom_messages.properties:49]: done

File: app/src/main/resources/static/js/passwordMeter.js [hunk 2]
```diff
@@ -0,0 +1,176 @@
+/**
+ * https://github.com/apereo/cas/blob/master/support/cas-server-support-thymeleaf/src/main/resources/static/js/passwordMeter.js
+ */
```
[the remark this answers is not in the document]
Comment (sklein94) [reply app/src/main/resources/static/js/passwordMeter.js:4]: done

File: app/src/main/resources/static/js/cas.js [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+            action = location.href;
+        } else {
+            action += location.search + encodeURIComponent(location.hash);
+        }
+        $('#fm1').attr('action', action);
+
+    });
+}
+
+function preventFormResubmission() {
```
[the remark this answers is not in the document]
Comment (sklein94) [reply app/src/main/resources/static/js/cas.js:92]: done

File: docs/development/Setup_Integrationtests_de.md [hunk 4]
```diff
@@ -42,9 +42,9 @@ module.exports = defineConfig({
             "ClientID": "inttest",
             "ClientSecret": "integrationTestClientSecret",
             "PasswordHintText": "Contact your admin",
-            "PrivacyPolicyURL": "https://www.triology.de/",
-            "TermsOfServiceURL": "https://www.itzbund.de/",
-            "ImprintURL": "https://cloudogu.com/"
+            "PrivacyPolicyURL": "https://www.google.de",
+            "TermsOfServiceURL": "https://www.google.de",
+            "ImprintURL": "https://www.google.de"
```
[the remark this answers is not in the document]
Comment (sklein94) [reply docs/development/Setup_Integrationtests_de.md]: by accident

**Review by kahoona77 — APPROVED** [review]
LGTM

**Pull request merged**